bin/testEnvLocalK8S.sh

```diff
@@ -25,7 +27,7 @@ export MASTER_CLUSTER_IP=10.99.0.1
 # TODO: customize the ports and generate a local config
 export KUBECONFIG=${TOP}/src/istio.io/istio/.circleci/config
 
-${ISTIO_GO}/bin/init.sh
+# make init
```

Comment:
commented out ? is that to avoid a loop as testEnvLocalK8S.sh is also invoked by make ?

Reply:
I wanted to see what, if anything, depended on this invocation.  This file's getDeps() function also calls init, so it seems like either 1) getDeps() doesn't need to call init since this script does it here on each invocation, or 2) whatever is dependent on this call to init (and ideally there'd be a comment here saying what was needed) can call init for itself or call this script with "ensure".  I can add this back (or address) if/when failures show up in testing, or just add back before the merge if people would rather play safe than sorry.

Reply:
there is no more dep at all anywhere needed now that we vendored